geodata-retriever/geodataRetriever.py

```diff
@@ -181,6 +181,10 @@ def UpdateFeatureClass(file, fcName):
             for file in os.listdir(os.path.join(saveFolder, d)):
                 if file.endswith('.shp'):
                     files.append(os.path.join(saveFolder, d, file))
+            if len(files) > 1:
+                print("Cannot handle zip folder with multiple shapefiles.")
+                logging.error("Cannot handle zip folder with multiple shapefiles.")
+                SendEmail('Geodata Retriever failed', 'Cannot handle zip folder with multiple shapefiles.')
```

Comment:
I think you're getting closer on this one ... by the looks of it, right now it will log the error, but then the script will keep going right? Since the script can't handle more than one shapefile it should just force exit when this scenario is encountered.